### PR TITLE
Add fontbakery shell script

### LINKED ISSUE #104: Add a shell script that runs all fontbakery checks
Let's call it `run_fontbakery.sh`

### PR DESCRIPTION
**Pull request by martelahaie**
This fixes #104

### CODE REVIEW

**Review by justvanrossum — COMMENTED** [review]
Hm, even if I install fontbakery, I get a bunch of FAILs related to missing dependencies. Please update requirements.txt to include fontbakery, but also any additional packages. For example, "dehinter" missing caused a FAIL, but when I installed it, another one popped up.

File: run_fontbakery.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+mkdir -p build/fontbakery/
```
Comment (justvanrossum) [inline run_fontbakery.sh:1]:
```suggestion
#!/bin/sh

set -e  # make sure to abort on error
set -x  # echo commands

mkdir -p build/fontbakery/
```

And please run `chmod +x run_fontbakery.sh` and commit that change. Without this we can't run it as `./run_fontbakery.sh`

**Comment by martelahaie** [comment]
Do you mean ERRORs? These popped up after installing the newest version of fontbakery, so I'm thinking it's rather a fontbakery issue than a problem with our build script, but I'll check.

File: run_fontbakery.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+set -e  # make sure to abort on error
+set -x  # echo commands
+
+mkdir -p build/fontbakery/
+
+for folder in Bungee_Basic Bungee_Layers Bungee_Rotated Bungee_Color
+do
+    for ttf in build/fonts/$folder/*.ttf
+    do
+		fontbakery check-googlefonts \
```
Comment (justvanrossum) [inline run_fontbakery.sh:12]: Please re-indent with spaces.

**Pull request merged**